runtime/development/src/lib.rs

```diff
@@ -1803,9 +1908,10 @@ pub type SignedExtra = (
 );
 /// Unchecked extrinsic type as expected by this runtime.
 pub type UncheckedExtrinsic =
-	generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
+	fp_self_contained::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 /// Extrinsic type that has already been checked.
-pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
+pub type CheckedExtrinsic =
+	fp_self_contained::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra, H160>;
```

Comment:
Would still be possible to validate calls the non-evm way? I am wondering on changes needed on clients like GSRPC

Reply:
`fp_self_contained` is pretty much a transparent wrapper that just exposes the necessary trait impls for handling EVM calls in the runtime API. I looked into the code to see if it would break SCALE-encoded compatibility with the existing Extrinsic/Call structures, and AFAICT it's safe

Reply:
Cool, and I assume the extrinsic payload to be signed is the same? Or a client would need to form the extrinsic to be signed differently?